### PR TITLE
Add initial nuget documentation

### PR DESCRIPTION
**Pull request by stevemchugh**
Please review @taharrison @petefitz @westonized @Gonzales

### CODE REVIEW

File: nuget.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+The website is protected by Windows Authentication, so you will need to provide your domain username and password to log on. If you go to the site from inside the Burrows network on Internet Explorer it should authenticate you automatically.
+
+## Publishing Packages
+
+You can either publish via the nuget.exe application
+
+> nuget push {package file} -s http://nuget.burrows.biz/ {apikey}
+>
+> API Key for publishing packages: **DFDDC5D8-AFAF-4F1B-8CD6-C4D6A555A741**
```
Comment (westonized) [inline nuget.md]: The Style guide is public. API key is now public.

Comment (westonized) [inline nuget.md]: Suggest we change the API key now and just explain how to find it on here.

Comment (stevemchugh) [inline nuget.md]: As agreed I will move that to the readme.md on http://tfs.burrows.biz/tfs/NuGet/_git/nuget.burrows.biz

**Comment by westonized** [comment]
LGTM, but note convo on slack about making nuget repo private

**Comment by westonized** [comment]
Maybe StyleGuide is not the place for this document

**Comment by taharrison** [comment]
+1: Maybe StyleGuide is not the place for this document

This merge is not about style, nor coding standards.

Network paths, api description etc; this is increasing the visible attack surface of Burrows.

Even if all these are locked down now, we are inviting future burrows users to add more similar information to this repo - what happens when an unsecure URL gets committed here - or what about our passwords?  "Deploying to azure is easy... just log in as XXX with password YYY"

We don't even need to accept the merge request.  By that point the text is visible permanently.

**Comment by taharrison** [comment]
Another example:
"You will need to be a member of the XXX group to be able to do this"

So if someone penetrates our network, you're giving them the names of the user groups they want to be added into to start attacking the endpoints they've gleaned from this?

**Pull request merged**